### PR TITLE
feat: add APIs of board microservice

### PR DESCRIPTION
**Pull request by stat-kwon**
related issue: #277

### CODE REVIEW

**Review by jihyungSong — APPROVED** [review]
LGTM.
confirmation plz for cross-check.

File: proto/spaceone/api/board/v1/post.proto [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  // is_required: true
+  string domain_id = 10;
+}
+
+message SendNotificationRequest {
+  // is_required: true
+  string board_id = 1;
+  // is_required: true
+  string post_id = 2;
+  // is_required: true
```
Comment (whdalsrnt) [inline proto/spaceone/api/board/v1/post.proto:87]: Change optional value

File: proto/spaceone/api/board/v1/post.proto [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  string domain_id = 3;
+}
+
+message PostRequest {
+  // is_required: true
+  string board_id = 1;
+  // is_required: true
+  string post_id = 2;
+  // is_required: true
```
Comment (whdalsrnt) [inline proto/spaceone/api/board/v1/post.proto:96]: Change optional value

File: proto/spaceone/api/board/v1/post.proto [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+}
+
+message GetPostRequest {
+  // is_required: true
+  string board_id = 1;
+  // is_required: true
+  string post_id = 2;
+  // is_required: false
+  repeated string only = 3;
+  // is_required: true
```
Comment (whdalsrnt) [inline proto/spaceone/api/board/v1/post.proto:107]: Change optional value

File: proto/spaceone/api/board/v1/post.proto [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  string board_id = 1;
+  // is_required: false
+  string post_id = 2;
+  // is_required: false
+  string category = 3;
+  // is_required: false
+  string writer = 4;
+  // is_required: false
+  string user_id = 5;
+  // is_required: true
```
Comment (whdalsrnt) [inline proto/spaceone/api/board/v1/post.proto:122]: Change optional value

**Review by whdalsrnt — CHANGES_REQUESTED** [review]
Change description

File: proto/spaceone/api/board/v1/post.proto (hunk 4 above)
Comment (stat-kwon) [reply proto/spaceone/api/board/v1/post.proto]: I have edited the part you mentioned.

**Comment by stat-kwon** [comment]
I have edited the part you mentioned.

File: proto/spaceone/api/board/v1/post.proto [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+  string contents = 4;
+  // is_required: false
+  PostOptions options = 5;
+  // is_required: false
+  string writer = 6;
+  // is_required: false
+  string domain_id = 10;
+  // is_required: true
+  string user_id = 11;
```
Comment (whdalsrnt) [inline proto/spaceone/api/board/v1/post.proto:60]: Remove user_id field.
The user_id field comes from metadata, not from passing parameters.

**Review by whdalsrnt — CHANGES_REQUESTED** [review]

**Review by whdalsrnt — APPROVED** [review]

**Pull request merged**